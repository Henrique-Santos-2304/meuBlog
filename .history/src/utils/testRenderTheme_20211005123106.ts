import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children: React.ReactNode): RenderResult => {
  return render(<ThemeProvider>{children}</ThemeProvider> );
};
