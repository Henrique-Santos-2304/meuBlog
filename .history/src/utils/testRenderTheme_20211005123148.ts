import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children: React.ReactElement): RenderResult => {
  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider> );
};
