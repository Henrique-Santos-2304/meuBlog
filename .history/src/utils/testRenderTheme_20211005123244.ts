import { ThemeProvider } from "styled-components";
import { render, RenderResult } from "@testing-library/react";
import theme from "../styles/theme";
export const renderTheme = (children) => {
  return render(<ThemeProvider theme={theme}>{children}</ThemeProvider> );
};
