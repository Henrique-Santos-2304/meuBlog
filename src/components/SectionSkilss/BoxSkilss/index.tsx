import * as S from "./styles";
import { imagesProps } from "types/typesComponents/typesSlide";

type boxSkilssProps = { data: imagesProps; marginBox: string };

const BoxSkilss = ({ data, marginBox }: boxSkilssProps) => (
  <S.ContainerSkilss aria-label={data.ariaLabel} marginBox={marginBox}>
    <S.ContentImage>
      <S.ImageSkilss
        src={data.image.url}
        alt={data.image.alternativeText}
        layout="fill"
      />
    </S.ContentImage>
    <S.TitleSkilss>{data.title}</S.TitleSkilss>
  </S.ContainerSkilss>
);

export default BoxSkilss;
