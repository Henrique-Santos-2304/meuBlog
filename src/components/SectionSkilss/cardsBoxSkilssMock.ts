export const images1 = [
  {
    title: "Typescript",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679004/typescript_c5xdpy_24c4f9216e.svg",
  },
  {
    title: "JavaScript",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634678981/javascript_icon_zaobje_684ef7d6e9.svg",
  },
  {
    title: "React",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679023/react_eugvgw_ab3e29360c.svg",
  },
  {
    title: "Next",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679043/nextjs_jbhm7d_cba71019e8.svg",
  },
  {
    title: "Jest",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679060/jest_yvhqf3_956a129e31.svg",
  },
  {
    title: "Cypress",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679080/cypress_bmhnah_a9f894ff7d.svg",
  },
  {
    title: "Storybook",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679131/storybook_q1s5cl_dafe82d169.svg",
  },
];
export const images2 = [
  {
    title: "Typescript",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679004/typescript_c5xdpy_24c4f9216e.svg",
  },
  {
    title: "JavaScript",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634678981/javascript_icon_zaobje_684ef7d6e9.svg",
  },
  {
    title: "React",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679023/react_eugvgw_ab3e29360c.svg",
  },
  {
    title: "Next",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679043/nextjs_jbhm7d_cba71019e8.svg",
  },
  {
    title: "Jest",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679060/jest_yvhqf3_956a129e31.svg",
  },
  {
    title: "Cypress",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679080/cypress_bmhnah_a9f894ff7d.svg",
  },
  {
    title: "Storybook",
    img: "https://res.cloudinary.com/defnibbpl/image/upload/v1634679131/storybook_q1s5cl_dafe82d169.svg",
  },
];
