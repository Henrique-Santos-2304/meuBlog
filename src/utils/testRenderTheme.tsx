import { ThemeProvider } from "styled-components";
import { ChakraProvider } from "@chakra-ui/react";

import { render, RenderResult } from "@testing-library/react";
import theme from "styles/theme";
export const renderTheme = (children: React.ReactNode): RenderResult => {
  return render(
    <ThemeProvider theme={theme}>
      <ChakraProvider> {children}</ChakraProvider>
    </ThemeProvider>
  );
};
