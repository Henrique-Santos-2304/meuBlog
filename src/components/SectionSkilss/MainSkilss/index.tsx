import Headings from "components/globalComponents/Headings";
import SliderPhoto from "components/globalComponents/SliderPhoto";
import { images1, images2 } from "../cardsBoxSkilssMock";
import * as S from "./styles";

const SectionSkilss = () => {
  return (
    <>
      <S.ContentSkilss id="skilss">
        <Headings
          msg="Minhas Habilidades"
          colorEffect="#b7f600"
          colorText="#f8f8f8"
          fontSizeText="2rem"
          marginText="0 0 2rem 3rem"
          aria-label="Titulo da seção"
        />
        <SliderPhoto
          images={images1}
          receptorCards="skilss"
          slidesToShow={2.4}
          marginBox="2rem 1.5rem"
        />
        <S.Span></S.Span>
        <SliderPhoto
          images={images2}
          receptorCards="skilss"
          slidesToShow={2.4}
          marginBox="3rem 1.5rem 2rem"
        />
      </S.ContentSkilss>
    </>
  );
};

export default SectionSkilss;
