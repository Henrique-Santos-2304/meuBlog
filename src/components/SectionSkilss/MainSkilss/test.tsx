import "match-media-mock";
import SectionSkilss from ".";
import { screen } from "@testing-library/react";
import { renderTheme } from "utils/testRenderTheme";

describe("<SectionSkilss />", () => {
  it("should render ", () => {
    const { container } = renderTheme(<SectionSkilss />);
    expect(container).toMatchSnapshot();
  });
  it("should render ", () => {
    renderTheme(<SectionSkilss />);

    const title = screen.getByRole("heading", {
      name: /minhas habilidades/i,
    });

    expect(title).toBeInTheDocument();
  });
});
