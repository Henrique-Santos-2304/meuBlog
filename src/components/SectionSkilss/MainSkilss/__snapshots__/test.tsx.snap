// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionSkilss /> should render  1`] = `
.c1 {
  position: relative;
  width: -webkit-max-content;
  width: -moz-max-content;
  width: max-content;
  color: #f8f8f8;
  padding: 0;
  margin: 0 0 2rem 3rem;
  font-size: 2rem;
  font-family: "Roboto",monospace;
}

.c1::before {
  content: "";
  position: absolute;
  top: 50%;
  left: -1.5rem;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  width: 0.4rem;
  height: 90%;
  border-radius: 2rem;
  background: #b7f600;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: rgba(0,0,0,0.55);
  margin: 2rem 1.5rem;
  padding: 2rem;
  border-radius: 1rem;
  position: relative;
}

.c3::before {
  content: "";
  position: absolute;
  width: 0.3rem;
  height: 70%;
  top: 50%;
  right: -1.5rem;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  background: #b7f600;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: rgba(0,0,0,0.55);
  margin: 3rem 1.5rem 2rem;
  padding: 2rem;
  border-radius: 1rem;
  position: relative;
}

.c8::before {
  content: "";
  position: absolute;
  width: 0.3rem;
  height: 70%;
  top: 50%;
  right: -1.5rem;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  background: #b7f600;
}

.c4 {
  width: 5rem;
  height: 5rem;
  position: relative;
}

.c5 {
  margin-top: 1.3rem;
  font-size: 1.4rem;
  font-weight: 300;
  text-align: center;
  color: #f8f8f8;
}

.c2 {
  position: relative;
}

.c6 {
  bottom: -2.4rem;
}

.c6 ul:last-child {
  color: #b7f600;
}

.c6 ul li {
  color: #b7f600;
}

.c6 ul li.slick-active button::before {
  color: #b7f600;
}

.c6 ul li button::before {
  font-size: 2.3rem;
}

.c0 {
  margin: 2rem 0;
}

.c7 {
  margin: 2rem 0;
}

<div>
  <section
    class="c0"
    id="skilss"
  >
    <h1
      class="c1"
    >
      Minhas Habilidades
    </h1>
    <div
      class="slick-slider c2 slick-initialized"
      dir="ltr"
    >
      <div
        class="slick-list"
      >
        <div
          class="slick-track"
          style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
        >
          <div
            aria-hidden="false"
            class="slick-slide slick-active slick-current"
            data-index="0"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c3"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Typescript
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="false"
            class="slick-slide slick-active"
            data-index="1"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c3"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  JavaScript
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="false"
            class="slick-slide slick-active"
            data-index="2"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c3"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  React
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="3"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c3"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Next
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="4"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c3"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Jest
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="5"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c3"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Cypress
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="6"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c3"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Storybook
                </h3>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="c6 slick-dots"
      >
        <ul
          aria-label="Botões para trocar as fotos"
        >
           
          <li
            class="slick-active"
          >
            <button>
              1
            </button>
          </li>
          <li
            class=""
          >
            <button>
              2
            </button>
          </li>
          <li
            class=""
          >
            <button>
              3
            </button>
          </li>
          <li
            class=""
          >
            <button>
              4
            </button>
          </li>
          <li
            class=""
          >
            <button>
              5
            </button>
          </li>
          <li
            class=""
          >
            <button>
              6
            </button>
          </li>
           
        </ul>
      </div>
    </div>
    <div
      class="c7"
    />
    <div
      class="slick-slider c2 slick-initialized"
      dir="ltr"
    >
      <div
        class="slick-list"
      >
        <div
          class="slick-track"
          style="opacity: 1; transform: translate3d(0px, 0px, 0px);"
        >
          <div
            aria-hidden="false"
            class="slick-slide slick-active slick-current"
            data-index="0"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c8"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Typescript
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="false"
            class="slick-slide slick-active"
            data-index="1"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c8"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  JavaScript
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="false"
            class="slick-slide slick-active"
            data-index="2"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c8"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  React
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="3"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c8"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Next
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="4"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c8"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Jest
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="5"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c8"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Cypress
                </h3>
              </div>
            </div>
          </div>
          <div
            aria-hidden="true"
            class="slick-slide"
            data-index="6"
            style="outline: none; width: 0px;"
            tabindex="-1"
          >
            <div>
              <div
                aria-label="Linguagens,Frameworks ou bibliotecas dominadas"
                class="c8"
              >
                <div
                  class="c4"
                >
                  <div
                    style="display: block; overflow: hidden; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; margin: 0px;"
                  >
                    <img
                      class=""
                      data-nimg="fill"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </div>
                </div>
                <h3
                  class="c5"
                >
                  Storybook
                </h3>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="c6 slick-dots"
      >
        <ul
          aria-label="Botões para trocar as fotos"
        >
           
          <li
            class="slick-active"
          >
            <button>
              1
            </button>
          </li>
          <li
            class=""
          >
            <button>
              2
            </button>
          </li>
          <li
            class=""
          >
            <button>
              3
            </button>
          </li>
          <li
            class=""
          >
            <button>
              4
            </button>
          </li>
          <li
            class=""
          >
            <button>
              5
            </button>
          </li>
          <li
            class=""
          >
            <button>
              6
            </button>
          </li>
           
        </ul>
      </div>
    </div>
  </section>
</div>
`;
